package.json

```diff
@@ -12,6 +12,7 @@
 		"fix:prettier": "prettier --write '**/*.{json,yml,yaml}'",
 		"fix:eslint": "eslint --fix --ext .js,.ts,.vue --ignore-path .gitignore . --ignore-pattern 'cypress'",
 		"fix:stylelint": "stylelint --fix '**/*.{css,scss,vue}'",
+		"netlify": "netlify",
```

Comment:
I wonder if this is worth adding… I’d probably just use `npx netlify` instead of `npm run netlify` most of the time, since it’s shorter. Do we want to put that in the README instead?

Reply:
what would the difference be between the two? :eyes:

Reply:
No difference, as far as I’m aware. One’s just shorter than the other.

Reply:
if I don't add the script i can't run netlify with npm, unless i specify the full path to the bin in `node_modules`. I would prefer to leave the npm command because i think it is more used that `npx` and easier to understand for other people looking at the code

Reply:
Well, you can also use `npm x netlify` – `npx` is just a shortcut for `npm x` (= `npm exec`), if I understand correctly. But if you want to keep this one, I don’t mind either.

Reply:
Yes, if the user knows how to use the `npx` command I guess they can also figure that one out and decide if they want to use `npm` or `npx`. [I see that a difference is that with npx](https://www.freecodecamp.org/news/npm-vs-npx-whats-the-difference/) you can use packages that are not installed locally, but we are installing netlify locally.